src/GovUk.Education.ExploreEducationStatistics.Admin/Services/PublicationService.cs

```diff
@@ -184,7 +190,15 @@ public class PublicationService : IPublicationService
             return await _persistenceHelper
                 .CheckEntityExists<Publication>(publicationId, HydratePublicationForPublicationViewModel)
                 .OnSuccess(_userService.CheckCanViewPublication)
-                .OnSuccess(publication => _mapper.Map<PublicationViewModel>(publication));
+                .OnSuccess(async publication =>
+                {
+                    var publicationViewModel = _mapper.Map<PublicationViewModel>(publication);
+                    var methodologies = await _methodologyRepository.GetLatestByPublication(publicationId);
```

Comment:
I wasn't sure whether to do it like this where we defer to the repository to only fetch the latest methodologies for the publication, or keep the hydrate method like it was which loaded all methodologies/versions into the publications, and use automapper to only map the latest version like we do for Publication->MyPublicationViewModel.